test/integration/component/test_tags.py

```diff
@@ -2543,3 +2543,248 @@ def test_24_public_IP_tag(self):
             "List tags should return empty response"
         )
         return
+    @attr(tags=["advanced"], required_hardware="false")
```

Comment:
Do we really need hardware to test this, @yvsubhash ?

Reply:
@DaanHoogland  all the tests in test_tags moved to smoke as they all run on simulator